pkg/storage/utils/decomposedfs/tree/tree.go

```diff
@@ -238,12 +236,19 @@ func (t *Tree) GetMD(ctx context.Context, n *node.Node) (os.FileInfo, error) {
 // CreateDir creates a new directory entry in the tree
 func (t *Tree) CreateDir(ctx context.Context, n *node.Node) (err error) {
 
-	if n.Exists || n.ID != "" {
+	if n.Exists {
 		return errtypes.AlreadyExists(n.ID) // path?
 	}
 
+	// Allow passing in the node id
```

Comment:
spureous?

Reply:
Removed in https://github.com/cs3org/reva/pull/2383